cmd/README.md

```diff
@@ -0,0 +1,639 @@
+# Precompile Generation Tutorial
+
+We can now generate a stateful precompile with the Precompile gen tool!
```

Comment:
Can we improve this introduction to briefly explain what a stateful precompile is, what the precompilegen tool does, and what users will do in this tutorial?

Reply:
We should make it clear that this will be a walk through of creating a simple HelloWorld stateful precompile

Reply:
Ah sounds good